apps/example-orchestrator-ui/components/Metadata/Products/Products.tsx

```diff
@@ -79,10 +83,27 @@ export const Products: FC<ProductsProps> = ({
         status: {
             field: PRODUCT_FIELD_STATUS,
             name: COLUMN_LABEL_STATUS,
+            width: '90',
+            render: (value) => (
+                <WFOStatusBadge status={value.toLocaleLowerCase()} />
+            ),
         },
         productBlocks: {
             field: PRODUCT_FIELD_PRODUCT_BLOCKS,
             name: COLUMN_LABEL_PRODUCT_BLOCKS,
+            render: (productBlocks) => {
```

Comment:
you don't need the function brackets when only having a return inside `(productBlocks) => (<></>)`. same with the function inside of this.

Reply:
It's my personal preference to always include explicit returns. If you have to add something later - or temporarily like a console.log - it's easier. I'm as flexible though but I think we should probably make it a rule: 
https://eslint.org/docs/latest/rules/arrow-body-style, "as-needed" ?

Reply:
Good to make it a rule and you can always add the brackets when it becomes more then just a return.

Reply:
made the rule and enforced it